user-service/src/main/java/rs/edu/raf/banka1/controller/UserController.java

```diff
@@ -159,6 +154,34 @@ public ResponseEntity<ActivateAccountResponse> activateAccount(@PathVariable Str
         return new ResponseEntity<>(userService.activateAccount(token, password), HttpStatus.OK);
     }
 
+    @PostMapping(value = "/reset/{email}")
+    @Operation(summary = "Reset password", description = "Send password reset email to user")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "Email with password reset URL sent successfully",
+                    content = {@Content(mediaType = "application/json",
+                        schema = @Schema(implementation = ActivateAccountResponse.class))}),
+            @ApiResponse(responseCode = "500", description = "Internal server error")
+    })
+    public ResponseEntity<Void> sendResetPasswordEmail(@PathVariable String email) {
```

Comment:
Since your method sendResetPasswordEmail returns the value of method emailService.sendEmail (true if success, false if not), you can return HTTP status code from controller based on that value. HTTP.OK if sendResetPasswordEmail returned true, HttpStatus.INTERNAL_SERVER_ERROR otherwise.